debian/postinst

```diff
@@ -23,7 +23,6 @@ fi
 
 case "$1" in
     configure)
-        service apache2 reload || true
```

Comment:
Do you still need the surrounding `case` block? It doesn't do anything anymore!

Reply:
I was leaving that in as people tend to forget to re-add it if they make changes. Without it things can execute multiple times on one install.